tools/cli/util.go

```diff
@@ -927,6 +927,19 @@ func paginate(c *cli.Context, paginationFn collection.PaginationFn) error {
 	return nil
 }
 
+func mapToEnumValue(search string, candidates map[string]int32) (int32, error) {
```

Comment:
I would add special case:
```
if search == "" {
    return 0, nil
}
```
because empty string is logically equal to `Unspecified`.